Pokedex/src/components/PokeDesc/styles.ts

```diff
@@ -178,4 +176,49 @@ export const HomePageButton = styled.button`
     color: #373299;
     cursor: pointer;
     font-size: 14px;
+`;
+
+export const LocationContainer = styled.div`
+    display: flex;
+    flex: 1;
+    gap: 70px;
+    align-items: center;
+
+    @media (max-width: 500px) {
+        gap: 10px;
+    }
+`;
+
+export const IconsContainer = styled.div<{ $isActive: boolean }>`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 30px;
+    margin-top: 20px;
+    background: #F7F7F9;
+    box-shadow: 2px 2px 4px 0px #00000026;
+    width: 50px;
+    height: 50px;
+    border-radius: 10px;
+
+    &:active {
+        background: #e0e0e0;
+    }
+
+    ${({ $isActive }) => $isActive && `
+        background: #686188;
+    `}
```

Comment:
Just so you know, it's a must. All colors stay in COLORS const. If the designer changes the color view, you need to change it in one place.